bsip-0056.md

```diff
@@ -0,0 +1,314 @@
+exchanges. Centralized exchanges hold customer funds in custody and allow trading within their own (closed) platform
+providing order books, order matching and fund management (deposits and withdrawals). In contrast, decentralized
+exchanges put the order book and order matching into the blockchain which offers increased transparency. Additionally,
+no 3rd party needs to hold customer funds in custody and traders have a more direct access to the markets being able to
+place orders into the order book directly.
+
+Both approaches provide advantages and disadvantages. While centralized exchanges allow for high-frequency trading (HFT)
+and sophisticated trading strategies (partially hidden from the order book), they come with the disadvantage of low
+transparency on the order matching algorithm, potential for front-running as well as theft and hacks that may damage
+many customers at the same time.
+
+In contrast, decentralized exchanges do not require a trusted third party to hold customer funds in custody and offer
+high transparency through the audit trail of the blockchain and autonomy of order matching as part of the "smart
+contracts" that execute the trading. Most blockchain technologies, however, have the disadvantage of high latency (bitcoin transaction are
+processed only once every 10 minutes on average, BitShares transactions are processed every 3 seconds) and thus do not
+allow for HFT. Additionally, sophisticated trading strategies cannot be applied as they require some degree of
+confidentiality (e.g. the price limit of stop-loss orders).
+
+The concept proposed in this document tries to benefit from both worlds while offering additional features to traders.
+This is done by filling the gap between centralized and decentralized exchanges.
+
+It is proposed to use the blockchain as settlement layer for off-chain trading platforms while removing the default risk
+of the centralized trading platform. While decentralized exchanges follow the concept "the trade is the settlement", we
+here propose to let trades happen outside the blockchain while settlements are performed regularly on-chain. 
+
+## Savings Account
+
+Managed vesting balances furthermore allow to setup *savings accounts* (or rather *savings balances*) which are balances
+that can only be accessed after a certain amount of time has elapsed (manager can be set optional).
+
+# Use-case off-chain activities
+
+We propose to extend the BitShares Blockchain in that each account is provided an additional freedom and possibilities 
+in configurating their vesting balances with locked up funds.
+
+## Exchange perspective
+
+### Customer sign up
+When a new customer signs up with the centralized, external exchange (CEX), a BitShares account must be uniquely linked. This can be done e.g. with a signed message. 
+
+### Customer on-boarding
+The centralized, external exchange (CEX) must monitor all vesting balances that are created on the BitShares Blockchain.
```

Comment:
Why must a CEX monitor all vesting balances and not just the vesting balances for which it is a manager?